packages/i18n/src/messages/localizedConstants.tsx

```diff
@@ -660,6 +660,12 @@ const poolCandidateStatuses = defineMessages({
     id: "yRxy3a",
     description: "The pool candidate's status is Qualified Withdrew.",
   },
+  [PoolCandidateStatus.PlacedTentative]: {
+    defaultMessage: "Offer in Progress",
```

Comment:
Should be sentence case (REF: https://github.com/GCTC-NTGC/gc-digital-talent/issues/9076#:~:text=EN%3A-,Offer%20in%20progress,-FR%3A%20Offre%20en)

French ID will also need to change based on different hash.
```suggestion
    defaultMessage: "Offer in progress",
```

Reply:
I'm going to push back, I was intentional with it. All the other statuses aren't sentence cased. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5eb888bc-b088-4979-9c3f-c772f97f2f42)

But the French ones, except one, are it seems. Going to need an issue to decide what we want 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/371457ab-e6ea-4489-bee4-913a335ff986)

Reply:
I think we should at least create a ticket to fix these. I'm pretty sure GC guidelines are that we always use sentence case rather than title case.

Reply:
Issue for reference https://github.com/GCTC-NTGC/gc-digital-talent/issues/9385